### PR TITLE
Support filtering bank feature map, StoreLinkingRequest, consent retrieval for alternative access

### PR DESCRIPTION
**Pull request by mahendra-aps**

### CODE REVIEW

File: core/src/Rpc/Client.cs [hunk 1]
```diff
@@ -736,19 +736,6 @@ public Task<IList<Device>> GetPairedDevices()
                 .ToTask(response => (IList<Device>)response.Devices);
         }
```
Comment (JugrajSripalJain) [inline core/src/Rpc/Client.cs:738]: I thought we were not using core project at all. Is that not correct? If yes, then why change anything?

Comment (JugrajSripalJain) [reply core/src/Rpc/Client.cs:738]: Why was this added and why is this deleted now?

File: tpp/src/Member.cs [hunk 2]
```diff
@@ -970,5 +970,70 @@ public Account CreateTestBankAccountBlocking(double balance, string currency)
...
+        /// Get the external metadata from the bank associated with a token request
+        /// </summary>
+        /// <param name="tokenRequestId">tokenRequestId token request ID</param>
+        /// <returns>external metadata</returns>
+        public ExternalMetadata GetExternalMetadataBlocking(string tokenRequestId)
+        {
+            return GetExternalMetadata(tokenRequestId).Result;
+        }
     }
 }
```
Comment (JugrajSripalJain) [inline tpp/src/Member.cs:1039]: new line

File: core/src/Rpc/Client.cs (hunk 1 above)
Comment (mahendra-aps) [reply core/src/Rpc/Client.cs:738]: > I thought we were not using core project at all. Is that not correct? If yes, then why change anything?

Core project is base project for Tpp and User projects. Tpp and User use Core as project dependency.

Comment (mahendra-aps) [reply core/src/Rpc/Client.cs:738]: > Why was this added and why is this deleted now?

With latest version of Gateway.proto, I don't see any message field for VerifyAffiliateRequest and VerifyAffiliateResponse. Neither this method is present in SDK java as well, so it was removed

File: tpp/src/Member.cs (hunk 2 above)
Comment (mahendra-aps) [reply tpp/src/Member.cs]: done

**Pull request merged**